### ISSUE TITLE
Inspector won't work with open command

### ISSUE THREAD

**Issue by Funfun** [teampoltergeist/poltergeist#579]
Running
```open //localhost:9494```
returns
```The file //localhost:9494 does not exist.```

i think Capybara::Poltergeist::Inspect#url should contains "http:" prefix for url method.

**Comment by bbuchalter** [comment]
This is happening for me as well. Running OS X, Phantom JS 2.0.

**Comment by edgarjs** [comment]
+1

**Comment by cptactionhank** [comment]
Experiencing the same problem

**Comment by dholdren** [comment]
I just ran into this myself:
```
Poltergeist execution paused. Press enter to continue.
The file //localhost:9664 does not exist.
```

**Comment by andreierdoss** [comment]
+1 Same issue here.

**Comment by Petercopter** [comment]
:+1: Weird, it was working the other day...

**Comment by karlingen** [comment]
+1

**Comment by yaauie** [comment]
I don't think [Capybara's DSL](https://github.com/jnicklas/capybara/blob/master/lib/capybara/session.rb) provides an `open` method, and I believe that your `open` is resolving to the method stdlib's openuri [adds to Kernel](http://ruby-doc.org/stdlib-1.9.3/libdoc/open-uri/rdoc/Kernel.html#method-i-open).

If this is the case, `open` is probably not the method you're looking for (maybe `visit`?).

To validate my claim, you can puts the method's source location by prepending the following line above your failing line, or by inspecting your exception's backtrack:

~~~ ruby 
$stderr.puts method(:open).source_location.join(':')
~~~

**Comment by edgarjs** [comment]
it's not the `open` method, that's provided by Launchy. It's the url syntax, it's missing the protocol and hence it cannot be recognized as an URL

**Comment by hooverlunch** [comment]
+1

**Comment by mmack** [comment]
+1

**Comment by imajes** [comment]
running into this also.

**Comment by nateberkopec** [comment]
I guess I understand that poltergeist doesn't want to just slap http:// on there because someone might be doing strict-transport SSL or something in development, but it seems unlikely. I'm not sure what another solution might be.

**Comment by meitalfed** [comment]
+1

**Comment by molson83** [comment]
+1

**Comment by nateberkopec** [comment]
Rather than continuing to +1 on this issue, feedback on the PR here would be helpful: #582

**Comment by Nuru** [comment]
v1.6.0 patched to fix this and #654 is available at
```
 gem 'poltergeist', github: 'Nuru/poltergeist', branch: '160-remote-debugging'
```

**Comment by route** [comment]
Should be fixed in master now
